docs-ui/stories/components/form-fields.stories.js

```diff
@@ -374,4 +373,4 @@ export const __TextCopyInput = () => (
   <TextCopyInput onCopy={action('Copied!')}>Value to be copied </TextCopyInput>
 );
 
-__TextCopyInput.storyName = 'TextCopyInput';
+__TextCopyInput.storyName = 'Text Copy';
```

Comment:
Should story name match the component name exactly?

Reply:
You can still look up TextCopy and it'll show you this story. However, if the story name is too different from the component name (which is currently a bit rare), then it won't show up.

I'll likely revise the Storybook sidebar. Among other things, it'll allow people to search for stories using metadata such as name, component name, and even related keywords (which will be custom-defined in the story files). Storybook allows for some overwriting of their default UI and behavior. I just have to find an efficient way to do it for the sidebar.